lib/notifications/receive.dart

```diff
@@ -149,7 +149,7 @@ class NotificationService {
         await addFcmToken(connection, token: token);
 
       case TargetPlatform.iOS:
-        const appBundleId = 'com.zulip.flutter'; // TODO(#407) find actual value live
+        final appBundleId = await ZulipBinding.instance.getAppBundleId(); // TODO(#407) find actual value live
```

Comment:
Then to access the value, we can do something like `await ZulipBinding.instance.packageInfo.packageName`

Reply:
I was thinking what if that pacakgeInfo variable is null?
What we can do instead is this ->

Future<String> getAppBundleId() async{
    if(syncPackageInfo !=null ){
      return syncPackageInfo!.packageName;
    }

    final packageInfo = await package_info_plus.PackageInfo.fromPlatform();
    return packageInfo.packageName;
  }

Reply:
> I was thinking what if that pacakgeInfo variable is null?

Yeah, good thought. I think we handled the same issue for `AboutZulipPage`. 

Checking the dartdoc, `packageInfo` can resolve to `null` if there was an error, but otherwise awaiting it should give us the needed information.

```dart
  /// Provides application package information,
  /// via package:package_info_plus.
  ///
  /// The returned Future resolves to null if an error is
  /// encountered while fetching the data.
  ///
  /// This wraps [package_info_plus.PackageInfo.fromPlatform].
  Future<PackageInfo?> get packageInfo;
```

Perhaps something like:
```suggestion
        final appBundleId = (await ZulipBinding.instance.packageInfo)?.packageName ?? 'some fallback value';
```

I'm not entirely sure what fallback value or error handling is appropriate here. Feel free to start a discussion in the [#mobile-dev-help channel](https://chat.zulip.org/#narrow/channel/516-mobile-dev-help) if you need help navigate this.

Reply:
Okay, this approach looks simple and subtle. I will adding the fallback value as 'com.zulip.flutter' for the binding class and 'com.zulip.flutter.test' for the test binding class.